Server/mods/deathmatch/logic/CPlayer.h

```diff
@@ -85,14 +85,14 @@ class CPlayer : public CPed, public CClient
     int                GetClientType() { return CClient::CLIENT_PLAYER; };
     unsigned long long GetTimeSinceConnected() { return m_ConnectedTimer.Get(); }
 
-    const char* GetNick() { return m_strNick; };
-    void        SetNick(const char* szNick);
+    const char* GetNick() { return m_strNick; }; // Todo: Return a const SString& here instead. Also, make this function a const
+    void        SetNick(const char* szNick); // Todo: change this to SString, then move SString into m_strNick. Move ctor needs to be added to SString..
```

Comment:
Please remove the TODO comments. IMHO it's very obvious this should be changed in a **CPlayer** refactor.

Reply:
That's why I added the todo, so that in can be changed in a separate PR.